src/main/resources/liquibase/changes-1.0.xml

```diff
@@ -28,4 +28,30 @@
          </column>
       </createTable>
    </changeSet>
+
+   <changeSet id="gs-2.1" author="rjulea">
+      <createTable tableName="genres">
+         <column name="id" type="bigint" autoIncrement="true">
+            <constraints primaryKey="true" nullable="false" uniqueConstraintName="users_pk"/>
+         </column>
+         <column name="public_identifier" type="${uuid.type}">
+            <constraints nullable="false" unique="true" uniqueConstraintName="gener_public_identifier_uk"/>
+         </column>
+         <column name="created_at" type="${timestamp.type}">
+            <constraints nullable="false"/>
+         </column>
+         <column name="last_updated_at" type="${timestamp.type}">
+            <constraints nullable="false"/>
+         </column>   
+         <column name="name" type="varchar(50)">
+            <constraints nullable="false"/>
+         </column>
+         <column name="popularity" type="bigint">
+            <constraints nullable="false"/>
+         </column>
+         <column name="description" type="varchar(50)">
```

Comment:
Maybe increase the dimension for the description if you want. You might have long sentences for some descriptions that might not fit in 50 letters.